csm_api_client/session.py

```diff
@@ -127,7 +134,7 @@ def save(self) -> None:
         if self.token is None:
             return
 
-        token = self.token
+        token = deepcopy(self.token)
```

Comment:
Why the `deepcopy`? I must be missing something.

Reply:
I wanted to avoid mutating `self.token` since it would be a reference to either `self.fetch_token` or `self.token_file_contents`. I guess it hasn't caused problems though so I can revert it.